elements/pfe-jump-links/src/pfe-jump-links.js

```diff
@@ -93,18 +93,20 @@ class PfeJumpLinksNav extends PFElement {
         if (this.getAttribute("sr-text")) {
           this.shadowRoot.querySelector("nav").prepend(div);
         }
-
+        if (this.hasAttribute("horizontal")) {
+          return;
+        }
         let html = "";
```

Comment:
If you set this up as:

```suggestion
        let html = "Jump to section";
```

You don't need the else statement and you can drop the `if(html)` query below. on line 107

Reply:
I think we do need in case there is no heading slot and it has the horizontal attribute. Both of the previous ifs would be false, and I think that text should still be there for assistive technology. Wdyt?

Reply:
It can't have the horizontal attribute because you exited the function on line 97 if that attribute existed.  If you initialize the html variable to the text and then reassign it if the slot is used, then you get it without needing the else on line 103.

Reply:
Let me know if you want to sync up about this - we can jump on a call.

Reply:
Ah, I get it. I was writing code thinking html would coerce to false, but setting it to a string initially removes that need.